js/data/odata/utils.js

```diff
@@ -446,45 +403,36 @@ function transformTypes(obj, options) {
             }
         }
     });
-}
-
-const serializeDate = function(date) {
-    return 'datetime\'' + formatISO8601(date, true, true) + '\'';
 };
 
-const serializeString = function(value) {
-    return '\'' + value.replace(/'/g, '\'\'') + '\'';
-};
+const serializeDate = date => `datetime'${formatISO8601(date, true, true)}'`;
 
-const serializePropName = function(propName) {
-    if(propName instanceof EdmLiteral) {
-        return propName.valueOf();
-    }
+const serializeString = value => `'${value.replace(/'/g, '\'\'')}'`;
 
-    return propName.replace(/\./g, '/');
-};
+export const serializePropName = propName =>
```

Comment:
Just a note. I think it's better to use brackets in multiline arrow functions. Here is an argumentation: https://github.com/airbnb/javascript#arrows--paren-wrap
Maybe we'll add this rule in future.

Reply:
I think we should discuss this rule on meeting.